Source/DafnyCore/DafnyMain.cs

```diff
@@ -203,5 +205,95 @@ public class DafnyMain {
       }
     }
 
+    public static void WarnAboutSuspiciousDependencies(DafnyOptions dafnyOptions, ErrorReporter reporter, ProofDependencyManager depManager) {
```

Comment:
I would put this code in a separate file. I think ideally this could would live in a plugin that could live in a separate assembly that hooks into some `PostSymbolVerification` event. Not needed for now but if we can make it look more like that I'd appreciate it.

Reply:
Will do.